.dotnet.azure/sdk/openai/tools/TestFramework/src/OpenAI.TestFramework.csproj

```diff
@@ -18,7 +18,9 @@
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Moq" />
     <PackageReference Include="System.ValueTuple" />
-    <PackageReference Include="System.ClientModel" VersionOverride="1.1.0-beta.5" />
+    <PackageReference Include="System.ClientModel" VersionOverride="1.1.0" />
```

Comment:
If you updated to 1.1.0 in Packages.Data.Props, I think you no longer need the version override?

Reply:
Unfortunately, the way the automatic package references are different for what is considered a client library vs what is considered test code. For System.ClientModel, we currently have a package reference version for client libraries, but none for test code. Since this is test code, I added the override here so it continues to compile.

At some point, I think the package reference for System.ClientModel should be added for test code in Packages.Data.props, but since that could affects things globally, I've avoided that for now.